api/src/main/java/io/strimzi/api/kafka/model/kafka/listener/KafkaListenerAuthentication.java

```diff
@@ -41,6 +42,7 @@ public abstract class KafkaListenerAuthentication implements UnknownPropertyPres
 
     @Description("Authentication type. " +
             "`oauth` type uses SASL OAUTHBEARER Authentication. " +
+            "`serviceaccount-oauth` type uses SASL OAUTHBEARER Authentication. " +
```

Comment:
Same as earlier -> maybe you can explain how it differs from `oauth`.

Reply:
Your proposed string would work here as well. In order to be slightly more descriptive maybe we could say something like: 'uses SASL OAUTHBEARER Authentication pre-configured with Kubernetes ServiceAccount token file location'

Reply:
Yeah, I think what you suggest would work fine here.